libs/core/src/components/pds-input/docs/pds-input.mdx

```diff
@@ -113,8 +113,82 @@ When `invalid` is true, the input displays the error message and provides a visu
   <pds-input component-id="pds-input-error-example" label="Email address" type="email" error-message="Please provide a valid email address" value="Frank Dux" invalid></pds-input>
 </DocCanvas>
 
-## References
+### Prefix and Suffix
+A prefix or suffix can be added to the input field to provide additional context or formatting. These elements are placed within the input container, directly adjacent to the input field.
+
+<DocCanvas client:only
+  mdxSource={{
+    react: '<PdsInput componentId="pds-input-prefix-suffix-example" label="Name" type="email" value="Frank Dux"><pds-icon name="mail" slot="prefix"></pds-icon><pds-button class="pds-input__suffix" variant="unstyled" slot="suffix"><pds-icon name="remove-circle"></pds-icon></pds-button></PdsInput>',
+    webComponent: '<pds-input component-id="pds-input-prefix-suffix-example" label="Name" type="email" value="Frank Dux"><pds-icon name="mail" slot="prefix"></pds-icon><pds-button class="pds-input__suffix" variant="unstyled" slot="suffix"><pds-icon name="remove-circle"></pds-icon></pds-button></pds-input>'
+  }}>
+  <pds-input component-id="pds-input-prefix-suffix-example" label="Name" type="email" value="Frank Dux">
+    <pds-icon name="mail" slot="prefix"></pds-icon>
+    <pds-button class="pds-input__suffix" variant="unstyled" slot="suffix"><pds-icon name="remove-circle"></pds-icon></pds-button>
```

Comment:
You could use the `icon-only` and `icon` props instead of the `pds-icon` for the content. This will allow for adding text within the button as well.

```
<pds-button class="pds-input__suffix" variant="unstyled" slot="suffix" icon-only icon="remove-circle">Remove</pds-button>
```